addons/binding/org.openhab.binding.dlinksmarthome/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/

```

Comment:
It should now, it was recently fixed for all bindings: https://github.com/openhab/openhab2-addons/pull/2179